README.md

```diff
@@ -1 +1,111 @@
-# other-hash-methods-code-along
+# Higher Level Hash Methods
+
+##Objectives
+* Learn some interesting methods we can call on hashes
+
+-
+
+Previously, you were asked to iterate over a hash and collect the key that pointed to the lowest value. We asked you not to use some of the higher level hash methods there. Now, we're going to learn a few tricks that can make a task like that much easier. 
+
+### `.values`
+
+You can collect all of the values in a hash with the `.values` method: 
+
+```ruby
+family_members = {mom: "Victoria", dad: "Richard", sister: "Zoe"}
+
+family_members.values
+#  => ["Victoria", "Richard", "Zoe"]
+```
+
+We can see that the `.values` method returns an array of the values of the keys in the hash. 
+
+### `.keys`
+
+This method, not surprisingly, returns an array containing all of the keys in the hash that `.keys` has been called on: 
+
+```ruby
+family_members = {mom: "Victoria", dad: "Richard", sister: "Zoe"}
+
+family_members.keys
+#  => [:mom, :dad, :sister]
+```
+
+### `.min`
+
+You can use the `.min` method on a hash to return the key/value *pair* that contains that **lowest** value: 
```

Comment:
Mention that it returns a `pair`....so what return type can return two different things? Arrays!